### ISSUE TITLE
increase aggresiveness for file downloads

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/undercrawler#39]
I think it makes sense to make file downloading more aggressive. Browsers allow ~6 parallel connections to the same server; it should be fine to download multiple static files in parallel from the same server. Currently we're throttling it to 1 parallel download on average using autothrottle algorithm. 

Autothrottle doesn't make much sense for static files anyways - file size can vary a lot; it doesn't make sense to adjust download delay for 10Kb JPEG file based on that we spent 1 minute downloading 50MB pdf file.

**Comment by lopuhin** [comment]
Yeah, i didn't realize the different size problem. Right now they just go to different slots.
